### PR TITLE
Chapter 10: Summary added

### PR DESCRIPTION
**Pull request by pefontana**

### CODE REVIEW

File: 10_bees_vs_wasps/10_bees_vs_wasps.jl [hunk 1]
```diff
@@ -341,6 +341,22 @@ md"
 We see that the accuracy got a little better now, reaching a maximum of $85\%$. For now this will be enough, but there are many things we can do to improve the performance of the model. Hopefully this chapter had given some starting point for you to dig deeper into these topic if it is of your interest!
 "
 
+# ╔═╡ c838ce7e-8562-11eb-12a9-4d5fed79fb02
+md"
+### Summary
+In this chapter, we implemented a convolutional neural network that discriminated between pictures of bees and wasps.
+
+First, we saw how neural networks work and explained their training process, where we set a loss function that indicates how our model is performing and feed it with labeled data so that their parameters find the right setting to optimize the loss function.
```
Comment (martinacantaro) [inline 10_bees_vs_wasps/10_bees_vs_wasps.jl:349]: replace: indicates
by: evaluates

Comment (martinacantaro) [reply 10_bees_vs_wasps/10_bees_vs_wasps.jl:349]: I think the clarity of this line can be improved. Maybe break it into two sentences, with a clear definition of loss function

File: 10_bees_vs_wasps/10_bees_vs_wasps.jl [hunk 2]
```diff
@@ -341,6 +341,22 @@ md"
 We see that the accuracy got a little better now, reaching a maximum of $85\%$. For now this will be enough, but there are many things we can do to improve the performance of the model. Hopefully this chapter had given some starting point for you to dig deeper into these topic if it is of your interest!
 "
 
+# ╔═╡ c838ce7e-8562-11eb-12a9-4d5fed79fb02
+md"
+### Summary
+In this chapter, we implemented a convolutional neural network that discriminated between pictures of bees and wasps.
+
+First, we saw how neural networks work and explained their training process, where we set a loss function that indicates how our model is performing and feed it with labeled data so that their parameters find the right setting to optimize the loss function. 
+Then we explained that convolutional neural networks are a particular kind of neural networks with the characteristic that their layers contain filters and have a great performance working with images.
```
Comment (martinacantaro) [inline 10_bees_vs_wasps/10_bees_vs_wasps.jl:350]: We learned that convolutional neural networks are a particular kind of neural networks where layers act as filters, and that they have have a great performance when working with images.

Comment (martinacantaro) [reply 10_bees_vs_wasps/10_bees_vs_wasps.jl:350]: @marianLambda

File: 10_bees_vs_wasps/10_bees_vs_wasps.jl [hunk 3]
```diff
@@ -341,6 +341,22 @@ md"
...
+
+Thus, we decided to create a simple convolutional neural network to classify the bees and wasps pictures.
```
Comment (martinacantaro) [inline 10_bees_vs_wasps/10_bees_vs_wasps.jl:352]: to classify our pictures of bees and wasps.

File: 10_bees_vs_wasps/10_bees_vs_wasps.jl [hunk 4]
```diff
@@ -341,6 +341,22 @@ md"
...
+We pre-processed our data, converting all images to gray scale and changing their resolution to keep the convolutional neural networks as simple as possible.
+To train the neural networks in an efficient way we divided the data in small batches and passed forward to the model in small groups.
```
Comment (martinacantaro) [inline 10_bees_vs_wasps/10_bees_vs_wasps.jl:354]: we divided the data in small batches which we fed to the model one by one.

File: 10_bees_vs_wasps/10_bees_vs_wasps.jl [hunk 5]
```diff
@@ -341,6 +341,22 @@ md"
...
+Then we plotted the accuracy and loss function of our model to see how it performed.
```
Comment (martinacantaro) [inline 10_bees_vs_wasps/10_bees_vs_wasps.jl:355]: to see how well it performed.

File: 10_bees_vs_wasps/10_bees_vs_wasps.jl [hunk 6]
```diff
@@ -341,6 +341,22 @@ md"
...
+Finally, we tried to improve our model, so we decreased the learning rate and trained again our model, obtaining better accuracy.
```
Comment (martinacantaro) [inline 10_bees_vs_wasps/10_bees_vs_wasps.jl:356]: , so we decreased the learning rate and trained it again,

**Review by martinacantaro — CHANGES_REQUESTED** [review]
